models/issues/pull.go

```diff
@@ -315,6 +316,29 @@ func (pr *PullRequest) LoadHeadRepo(ctx context.Context) (err error) {
 	return nil
 }
 
+// LoadRequestedReviewers loads the requested reviewers.
+func (pr *PullRequest) LoadRequestedReviewers(ctx context.Context) error {
+	if len(pr.RequestedReviewers) > 0 {
+		return nil
+	}
+
+	reviews, err := GetReviewersByIssueID(pr.Issue.ID)
```

Comment:
`GetReviewersByIssueID` this function name seems incorrect, it should be `GetReviewsByIssueID` ?

I know it is a legacy problem, but is it a good chance to rename it to a correct name?